pom.xml

```diff
@@ -204,6 +204,7 @@
     <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
     <jacoco-maven-plugin.version>0.7.6.201602180812</jacoco-maven-plugin.version>
     <mockito.version>1.10.19</mockito.version>
+    <session.timeout>720</session.timeout>
```

Comment:
Can you prefix this with "tomcat" and move it near the other tomcat property:

[https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218](https://github.com/zhx828/cbioportal/blob/999f892fae96698f154752875aad5fd1eaf7d572/pom.xml#L218)